engine/src/physics/solver/integration/physics_constants_integration.hpp

```diff
@@ -0,0 +1,24 @@
+/// @file
+/// @brief Resource @ref cubos::engine::PhysicsConstantsIntegration.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over constants in the integration plugin.
+    /// @ingroup physics-plugin
+    struct PhysicsConstantsIntegration
+    {
+        CUBOS_REFLECT;
+
+        float minInvMass = 0.0F;
+
+        float minInvInertia = 0.0F;
```

Comment:
Would be cool to document the meaning of these constants, maybe @fallenatlas can help on that